tools/sklearn/model_validation.xml

```diff
@@ -67,7 +68,16 @@ y = read_columns(
 y=y.ravel()
 
 options = params["model_validation_functions"]["options"]
-options['cv'] = get_cv( options['cv'] )
+splitter, groups = get_cv(options.pop('cv_selector'))
+if groups is not None:
+    groups = groups.strip()
+    if groups == "":
+        groups = None
```

Comment:
How does an input look like if this is true and why do we pass then None to the splitter ... is adding `  ` a actual usecase?

Reply:
It's consistent to cast empty string to `None` and may be helpful in debugging.

Reply:
Ok, if you think this is more readable. Should we use this as a macro?

Reply:
I'll look into this later, thanks.